solr/src/main/conf/schema.xml

```diff
@@ -0,0 +1,613 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more

```

Comment:
It is copied from a sample Apache Solr schema.xml file.

It's not actually required for the tarball at all; it's only there for users to make use and refer to as they configure their Solr servers separate from ycsb. So, would that make src/main/resources the best place for it then?